src/free-trial/tax/index.tsx

```diff
@@ -112,7 +112,7 @@ export const Tax: React.FC< TaxProps > = ( { onComplete, query, task } ) => {
 			updateAndPersistSettingsForGroup( 'tax', {
 				tax: {
 					...taxSettings,
-					wc_connect_taxes_enabled: 'no',
+					woo_tax_taxes_enabled: 'no',
```

Comment:
Currently, the Tax task uses `wc_connect_taxes_enabled` option to determine its completion status ([ref](https://github.com/woocommerce/woocommerce/blob/7dc8dd63bad9069307b095c05798a6eea6f40d1d/plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php#L112-L117)).

In order to support the new option, we'll need to override the default Tax task class, here's ab example of how we did it for Products task: https://github.com/Automattic/wc-calypso-bridge/pull/963/files.